### PR TITLE
Feature/hero banner

### LINKED ISSUE #55: Hero Banner
Hero banner used on https://www.888.de/888club/#sec3

### PR DESCRIPTION
**Pull request by kailasnadh790**
Fix #55 

## Test URLs
- https://feature-hero-banner--888de--aemsites.hlx.page/888club/

### CODE REVIEW

File: blocks/hero-banner/hero-banner.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    }
+
+    .hero-banner.block .hero-steps-wrapper {
+        flex-direction: row;
+        justify-content: center;
+    }
+}
+
+/* stylelint-disable-next-line media-feature-range-notation */
+@media screen and (max-width: 440px) {
```
Comment (helms-charity) [inline blocks/hero-banner/hero-banner.css:148]: can we use a min-width only? This is a request from David N's Collab best practices.

**Review by helms-charity — CHANGES_REQUESTED** [review]
Besides adding the div IDs in the body so we can test the anchor links, see my comment about the CSS media point

File: blocks/hero-banner/hero-banner.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+main .section.hero-banner-container>div {
+    margin: 0;
+    padding: 0;
+    max-width: unset;
+}
+
+
+.hero-banner.block {
```
Comment (dave-fink) [inline blocks/hero-banner/hero-banner.css:14]: Minor: Is .block necessary here - not sure that level of specificity is needed

Comment (kailasnadh790) [reply blocks/hero-banner/hero-banner.css:14]: ya..this is a convention we followed in my previous project.. to enforce specificity to that particular block.

**Review by dave-fink — APPROVED** [review]
LGTM

**Review by helms-charity — APPROVED** [review]
Looks good

**Pull request merged**